pathspider/plugins/ecnflags.py

```diff
@@ -92,70 +92,96 @@ def combine_flows(self, flows):
+            elif (flows[1]['tcp_synflags_rev'] & TCP_SAEC == TCP_SAE) &(flows[2]['tcp_synflags_rev'] & TCP_SAEC == TCP_SAE) &(flows[3]['tcp_synflags_rev'] & TCP_SAEC == TCP_SAE) & (flows[4]['tcp_synflags_rev'] & TCP_SAEC != TCP_SAE):
+                conditions.append("ecn.negotiation.ect.suceeded")
             
-                print("Key error in ecn_syn_rev: %s"%str(flows[i]['dip']))
+            elif (flows[1]['tcp_synflags_rev'] & TCP_SAEC == TCP_SAE) &(flows[2]['tcp_synflags_rev'] & TCP_SAEC != TCP_SAE) &(flows[3]['tcp_synflags_rev'] & TCP_SAEC != TCP_SAE) & (flows[4]['tcp_synflags_rev'] & TCP_SAEC != TCP_SAE):
+                conditions.append("ecn.negotiation.suceeded")
+                
+            elif (flows[1]['tcp_synflags_rev'] & TCP_SAEC != TCP_SAE) &(flows[2]['tcp_synflags_rev'] & TCP_SAEC != TCP_SAE) &(flows[3]['tcp_synflags_rev'] & TCP_SAEC != TCP_SAE) & (flows[4]['tcp_synflags_rev'] & TCP_SAEC != TCP_SAE):
+                conditions.append("ecn.negotiation.failed")
             
-        try:       
-            if flows[2]['tcp_synflags_rev'] & TCP_SAEC == TCP_SAE:
-                ect0success = "True"
-        except TypeError:
-            print("Type error in flow 2 %s"%str(flows[2]['tcp_synflags_rev']))
-            ect0success = "FAILFAIL1"
-        except KeyError:
-            print("Key error in flow 2")
-            ect0success = "FAILFAIL2"
-        try:
-            if flows[3]['tcp_synflags_rev'] & TCP_SAEC == TCP_SAE:
-                ect1success = "True"
-        except TypeError:
-            print("Type error in flow 3 %s"%str(flows[3]['tcp_synflags_rev']))
-            ect1success = "FAILFAIL1"
-        except KeyError:
-            print("Key error in flow 3")
-            ect1success = "FAILFAIL2"
-        try:
-            if flows[4]['tcp_synflags_rev'] & TCP_SAEC == TCP_SAE:
-                cesuccess = "True"
-        except TypeError:
-            print("Type error in flow 4 %s"%str(flows[4]['tcp_synflags_rev']))
-            cesuccess = "FAILFAIL1"
+            else:
+                conditions.append("ecn.negotiation.transient")
+        
         except KeyError:
-            print("Key error in flow 4")
-            ect1success = "FAILFAIL2"   
-                    
-
-                
-        conditions.append('ecn.connectivity.works %s/%s' %(work,len(flows)-1))
-        conditions.append('ecn.negotiation.succeeded %s/%s' %(success,len(flows)-1) )
-        conditions.append('ecn.ect0.success %s' %(ect0success))
-        conditions.append('ecn.ect1.success %s' %(ect1success))
-        conditions.append('ecn.ce.success %s' %(cesuccess))
-        conditions.append('ecn.ect0.rev.seen %s/%s' %(ect0,len(flows)-1) )
-        conditions.append('ecn.ect1.rev.seen %s/%s' %(ect1,len(flows)-1) )
-        conditions.append('ecn.ce.rev.seen %s/%s' %(ce,len(flows)-1) )
+            conditions.append("Not all flags observed")    
+#             if flows[i]['spdr_state'] == CONN_OK:
```

Comment:
Large chunks of commented code are not useful. This code will exist in the git history anyway. It's better to just delete it.